### ISSUE TITLE
How to re-enrich indexes once you modify Sorting Hat db

### ISSUE THREAD

**Issue by jsmanrique** [grimoirelab/training#4]
I think how to re-enrich the enriched indexes once you merge or modify profiles is missing in the Sorting Hat section

**Comment by jgbarah** [comment]
Yes, you're right. I still need to write about that. But before it, I wanted to make the guide on SortingHat itself a bit more complete...

I'm going to leave this open to not forget about reenriching indexes, anyway.
